### PR TITLE
Update Hannover's markets

### PR DESCRIPTION
**Pull request by katrinleinweber**
Based on manual comparison with "Stadt Hannover" list on
https://www.hannover.de/Veranstaltungskalender/M%C3%A4rkte/

### CODE REVIEW

File: cities/hannover.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
             "properties": {
                 "title": "Wochenmarkt Döhren",
                 "location": "Fiedelerplatz, 30519 Hannover",
-                "opening_hours": "Fr 8:00-13:00"
+                "opening_hours": "Di 14:00-18:00"
```
Comment (johnjohndoe) [inline cities/hannover.json:59]: @katrinleinweber Did you see the error in the build log:

```
===> Validating cities/hannover.json ...
HANNOVER: Wochenmarkt Döhren
Error: Di <--- (Please use the English abbreviation "Tu" for "di".)
Validation done. 0 warning(s), 1 error(s) detected.
```

**Comment by johnjohndoe** [comment]
Feel free to also update the data source and the time stamp as described in the contribution guide.

**Comment by johnjohndoe** [comment]
@katrinleinweber Did you receive my comment on your pull request? Do you have time to work on this?

File: cities/hannover.json (hunk 1 above)
Comment (katrinleinweber) [reply cities/hannover.json]: Sorry for the delay! Done now in bb4484ceaad8651c25f99501b07a6597d63e7ab6 :-)

**Review by johnjohndoe — APPROVED** [review]
Thank you for the update and the correction. :+1: It would be great if you keep them updated.
Please consider adding an update date which should be useful for visitors of the website.

**Comment by johnjohndoe** [comment]
@katrinleinweber Please also be welcome to promote the website in Hannover. There is a couple of options, please have a look at issue #38.

**Pull request merged**